spec/support/delayed_job.rb

```diff
@@ -2,6 +2,22 @@ module DelayedJobHelpers
   def run_all_delayed_jobs
     Delayed::Worker.new(exit_on_complete: true, quiet: true).start
   end
+
+  def most_recently_enqueued_job
+    Delayed::Job.asc('_id').last
```

Comment:
Very minor & subjective: I really like the intention-revealing methods that have been extracted, but ideally I wonder whether we need to access the `_id` attribute at all given that it feels like an implementation detail of Mongoid/MongoDB. I wonder whether we could use `created_at` or `updated_at` instead.